### PR TITLE
Fix multiple view creation

### PR DESCRIPTION
**Pull request by guerinoni**
This PR should fix issues #6

### CODE REVIEW

File: src/gui/MainWindow.cpp [hunk 1]
```diff
@@ -38,33 +38,24 @@ void MainWindow::selectFile()
...
+
     auto dock = ConfigViewFactory().makeConfigView(m_config->protocol);
+    dock->setObjectName("ConfigView");
     dock->setParent(this);
     this->addDockWidget(Qt::DockWidgetArea::LeftDockWidgetArea, dock,
                         Qt::Orientation::Vertical);
 }
 
-void MainWindow::create_menubar()
+void MainWindow::createMenubar()
```
Comment (marsiliano) [inline src/gui/MainWindow.cpp]: please align to camelCase createMenuBar

**Review by marsiliano — CHANGES_REQUESTED** [review]

File: src/gui/MainWindow.cpp (hunk 1 above)
Comment (guerinoni) [reply src/gui/MainWindow.cpp]: I'm aligned! This comment is inopportune

Comment (marsiliano) [reply src/gui/MainWindow.cpp]: The Item is a Menu Bar so to be compliant to camel case you should write menuBar, in your case it's createMenuBar

Please refer to those links:
https://en.wikipedia.org/wiki/Menu_bar
https://en.wikipedia.org/wiki/Camel_case

**Review by AndreaRicchi — APPROVED** [review]
**RTBM**!

**Pull request merged**